examples/010_harmonic_example.py

```diff
@@ -0,0 +1,241 @@
+"""
+.. _harmonic_example:
+
+Postprocess a harmonic analysis
+--------------------------------------
+
+.. note::
+
+    This example is work in progress. Please open a github issue if something is unclear.
```

Comment:
```suggestion
    This example is work in progress. Please open a Github issue if something is unclear.
```

Should we add a link to https://github.com/ansys/pydpf-composites/issues/new/choose ?